string name
float64 ledMode
float64 camMode
float64 pipeline
float64 stream
float64 snapshot